bin/install-boilerplate.js

```diff
@@ -66,7 +66,11 @@ const updatePackageJson = () => {
 const installPackages = () => {
     try {
       console.log('🍉 Installing packages ...');
-      cp.execSync('yarn install');
+      if (fse.existsSync('yarn.lock')) {
```

Comment:
I like this idea! 👍

Reply:
Thanks! However, after looking at it again with a fresh mind, I don't think this covers all cases.
If the user runs `poetic` immediately after `git init` and `yarn/npm init` their project dir will not contain a `yarn.lock` file :slightly_frowning_face:. A better solution would probably be to try `yarn install` first, then if that throws an error run `npm install`.

Reply:
Or provide an input similar to how CLI tools like `vue-cli` do and ask the user what their preference is...

Or provide a CLI argument to specific it... ;)

Honestly it seems odd to default to a dependency that isn't installed by default with node (or installed as a dep with the package in question.

Reply:
That's a good point, I agree. It should be customizable. Ideally, it should support a CLI argument, and if that is not provided it should ask what to use.

Reply:
@arianacosta when will this be available via npm, it was the one thing preventing me from using this and I am hoping to simplify a bit of boilerplate on some 100% npm driven projects.

Reply:
@ktiedt I was planning on releasing it this weekend, unfortunately, this doesn't work. This installs and all, but the functionality is broken. I've created another ticket for it: https://github.com/arianacosta/poetic/issues/28

I hope we can find a solution soon, but having NPM is going to be more challenging than what we originally thought. I'll keep you posted 👍